ThunderCats/UIAlertController+TCAdditions.h

```diff
@@ -27,12 +27,57 @@
 //
 
 #import <UIKit/UIKit.h>
+#import "TCNullabilityMacros.h"
 
 @interface UIAlertController (TCAdditions)
 
+typedef void (^ TCCancelAction)(UIAlertAction * __tc_nonnull);
```

Comment:
Be sure to add doc comments to typedefs.  A typedef defines a type in much the same way that classes, structs, and enums define a type.  They all equally deserve documentation.